index.html

```diff
@@ -51,6 +51,7 @@ <h2>Release Calendar</h2>
 			<tr style='font-family: Droid Sans Mono; font-size:80%' align="left" valign="top">
 				<td>
 					<ul>
+						<li><a href="/releases/mainnet/1.9.3/index.html">Release 1.9.3</a> (Friday, 2021.08.06)</li>
```

Comment:
Should rebase on latest changes (date format changed to uses `-` instead of `.` as the delimiter).

Reply:
done